frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/multiple-storageclass.ts

```diff
@@ -0,0 +1,72 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { DeviceSet } from '../../src/types';
+import { OCS_INTERNAL_CR_NAME } from '../../src/constants';
+import { NS } from '../consts';
+import { getCurrentDeviceSetIndex } from '../../src/utils/add-capacity';
+
+export const withJSONResult = (res: Cypress.Exec, scName: string, iAndD: IndexAndDeviceSet) => {
+  const jsonOut: K8sResourceKind = JSON.parse(res.stdout);
+  iAndD.deviceSets = jsonOut.spec.storageDeviceSets;
+  iAndD.index = getCurrentDeviceSetIndex(iAndD.deviceSets, scName);
+};
+
+export const fetchStorageClusterJson = () =>
+  cy.exec(`kubectl get --ignore-not-found storagecluster ${OCS_INTERNAL_CR_NAME} -n ${NS} -o json`);
+
+export const fetchWorkerNodesJson = () =>
+  cy.exec('oc get nodes -l "node-role.kubernetes.io/worker" -o json');
+
+export const addCapacity = (uid: string, scName: string) => {
+  console.log(`UID: ${uid} == SCName: ${scName}`);
```

Comment:
```suggestion
```
Debugging?

Reply:
ack... will remove it...
Actually, I did not add this statement... I just migrated the function to the different file...